com.unity.shadergraph/Editor/Generation/TargetResources/BlockFields.cs

```diff
@@ -1,56 +1,60 @@
-﻿using UnityEngine;

```

Comment:
Can probably revert this change

Reply:
Yeah, I don't even know how this happened or what's changed.
I think it's unity-meta and .editorconfig stuff.

What I did is just add CoatMask and CoatSmoothness BlockFields.

Reply:
I think line-ends changed.

Reply:
Yeah, I don't even know how this happened or what's changed, looks like line-ends.
I think it's .editorconfig stuff I've got set up.

What I did is just add CoatMask and CoatSmoothness BlockFields.